designsystem/src/commonMain/kotlin/org/hisp/dhis/mobile/ui/designsystem/component/InputField.kt

```diff
@@ -125,6 +128,11 @@ fun BasicTextField(
         handleColor = cursorColor,
         backgroundColor = Blue300,
     )
+
+    var textFieldSelection by remember {
```

Comment:
shouldn't update if the inputText changes?  `var textFieldSelection by remember(inputText) {...}`
What do you think?

Reply:
`onValueChange = {
                textFieldSelection = it.selection
                onInputChanged.invoke(it.text)
            }`
            This will update the selection when inputText changes, we don't need to do it in remember